docker-compose.yml

```diff
@@ -34,6 +34,7 @@ services:
   brunch:
     image: node:6.3.1
     command: node_modules/brunch/bin/brunch watch
+    restart: always
```

Comment:
True.

It was useful as we developed, and there were already some conflicting changes in master that would've led to a hard to test scenario. That prevented us from making this change in master and then merging it.

I should've kept it stashed.